composer/algorithms/cutout/cutout.py

```diff
@@ -12,7 +12,9 @@
 log = logging.getLogger(__name__)
 
 
-def generate_mask(mask: Tensor, width: int, height: int, x: int, y: int, cutout_length: int) -> Tensor:
+def generate_mask(mask: Tensor, width: int, height: int, x: int, y: int, cutout_length: Union[int, float]) -> Tensor:
+    if 0 < cutout_length < 1:
```

Comment:
The dual definition of `cutout_length` may be confusing. e.g. if 0 < 1 is a fraction, if >1 is an actual pixel. Since this is a helper function, I think its OK to keep this as pixels, and push the fraction vs pixel spec to the calling object.